lib/mix_test_interactive/interactive_mode.ex

```diff
@@ -38,10 +38,10 @@ defmodule MixTestInteractive.InteractiveMode do
   @doc """
   Process a command from the user.
   """
-  @spec process_command(String.t()) :: :ok | :quit
-  @spec process_command(GenServer.server(), String.t()) :: :ok | :quit
+  @spec process_command(String.t()) :: :ok
+  @spec process_command(GenServer.server(), String.t()) :: :ok
   def process_command(server \\ __MODULE__, command) do
-    GenServer.call(server, {:command, command})
+    GenServer.cast(server, {:command, command})
```

Comment:
I was wondering about a few things here:

* Should we differentiate this as a `process_command_async`?
* Are we able to queue up a command while another command is processing?

There are things I like about an automatic timeout using `call`. I wonder if we'd be able to handle the cli args in such a way to allow cancellation if the user realizes the run is taking too long. I'd also like to be careful about  allowing queuing up calls in the server during a long running test run.

Reply:
This is the only caller of `process_command` (other than tests), so IMO it doesn't need a different name.

Yes, we can queue up a command. The intention is to make this work the same as a shell in a terminal. If a long-running command is running, the shell will process any commands typed during the run once the command completes.  Given that analogy, I'm not worried about queueing up calls to the GenServer.

In terms of cancelling a test run, that's a feature I've thought about a bit, but haven't implemented yet.